sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -54,8 +55,11 @@ private[spark] object HiveUtils extends Logging {
     sc
   }
 
+  private val supportedHive2ShortVersions = Set("2.3.0")
+  val isSupportedHive2 = supportedHive2ShortVersions.contains(HiveVersionInfo.getShortVersion)
```

Comment:
Hive23? does this need to be a separate val?

Reply:
I don't think so. In the future, we could update `supportedHive2ShortVersions` to support Hive 2.4:
```scala
private val supportedHive2ShortVersions = Set("2.3.0", "2.4.0")
```

Reply:
Still, won't this not even match Hive 2.3.1-2.3.4? am I missing something basic?

Reply:
We use `shortVersion`, not the `version`:
![image](https://user-images.githubusercontent.com/5399861/55367974-32bce500-5522-11e9-8435-9f340abceb1a.png)
![image](https://user-images.githubusercontent.com/5399861/55367992-410b0100-5522-11e9-8cee-c4ab48b41cf1.png)

Reply:
It is confusing to use 2.3.0 here. Let us just match the prefix.

Reply:
OK